apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -103,19 +92,16 @@ export function ColumnToggleButton() {
             <FormControl>
                 <Select
                     multiple
-                    value={activeColumns}
+                    value={selectedColumnNames}
                     open={open}
                     onChange={handleChange}
                     onClose={handleClose}
                     renderValue={renderEmptySelectValue}
                     sx={{ visibility: 'hidden' }}
```

Comment:
This was recently committed to the main branch, but it's missing here.

```suggestion
                    sx={{ visibility: 'hidden', position: 'absolute' }}
```